.github/workflows/binary.yaml

```diff
@@ -0,0 +1,81 @@
+name: Binaries
+
+on:
+  schedule:
+    - cron: '0 1 * * *' # Triggers the build at 1:00 UTC time
+
+jobs:
+  binary:
+    name: Create
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-13
+            name: x86_64-apple-darwin
+            installable: .#
+          - os: macos-14
+            name: aarch64-apple-darwin
+            installable: .#
+          - os: ubuntu-22.04
+            name: x86_64-unknown-linux-musl
```

Comment:
I believe this should be `x86_64-unknown-linux-gnu` for compatibility with most linux distros. Or is it the case that muslc binaries work on gnu systems due to static linking or something? Even if this is the case, it might be worth it to provide gnu binaries.